ENVIRONMENT.md

```diff
@@ -0,0 +1,71 @@
+# Setting up your development environment
+
+The first time you run the test suite or any application using Log4JNA, Log4JNA attempts to write the necessary entries in 
+Windows<sup>TM</sup> Registry. If you don't have administrator privileges the run fails.
+
+### Running the tests
+In order for the tests to pass you must run them as Administrator or set up the registry before your first run.
+
+Open a command window as administrator and execute `mvn test`
+
+If you are using an IDE launch it as Administrator import the Maven projects and run the tests. 
+
+### IDE Setup
+
+Please be careful and do not check IDE configuration files into GitHub, you can add the project to your IDE following the instructions bellow.
+
+Eclipse configuration files are already added to `.gitignore`, please add any configuration file created by your IDE to the ignore file.
+
+#### Eclipse
+See [this article](https://books.sonatype.com/m2eclipse-book/reference/creating-sect-importing-projects.html) to import the Maven projects.
+
+#### IteliJ IDEA
+See [this article](https://www.jetbrains.com/help/idea/2016.1/importing-project-from-maven-model.html) to import the Maven projects.
+
+#### NetBeans
+See [this article](http://wiki.netbeans.org/MavenBestPractices) to import the Maven projects.
+
+### Set up the Registry manually

```

Comment:
This are just instructions on how to set up the development environment. 

I think that we should leave references to setting up the registry manually, it does not harm and might avoid questions from users.